src/features/views/components/ViewDataTable/columnTypes/OrganizerActionColumnType.tsx

```diff
@@ -27,6 +27,27 @@ import useToggleDebounce from 'utils/hooks/useToggleDebounce';
 
 type OrganizerActionViewCell = null | ZetkinOrganizerAction[];
 
+const sortByOa = (
+  v1: ZetkinOrganizerAction[],
+  v2: ZetkinOrganizerAction[]
+): number => {
+  const hasActions = (v: ZetkinOrganizerAction[]): boolean => {
+    return v.length > 0;
+  };
+
+  const getSortValue = (v: ZetkinOrganizerAction[]): number => {
+    if (!hasActions(v)) {
+      return 2;
```

Comment:
Why return 2? Sorting comparator functions should return a negative number, zero, or a positive number to indicate the relationship between the compared values (`v1` and `v2` in this case).

The size of the returned value does not matter (except zero), only whether it's negative or positive.

Returning 1 in some execution branches and 2 in other branches makes me think that it matters somehow, but it shouldn't, so that makes me think that it's a mistake. In any case, it makes me think, which makes reading more difficult. 😊 

Would it make sense to change this?

```suggestion
      return 1;
```